CMakeLists.txt

```diff
@@ -92,5 +92,4 @@ if(("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
 endif()
 
 add_subdirectory(src)
-
-install(FILES LICENSE DESTINATION .)
```

Comment:
I'm not about to remove `LICENSE` from installation, I mean we need to setup `CMakeLists` correctly.

Unfortunatelly, I am far from C++ project development and I have no way to master CMake. I'm just a stupid data scientist who promotes using [SciPy stack](http://www.scipy.org/about.html) in the work :(.